libcontainer/cgroups/fs/cpu.go

```diff
@@ -84,6 +84,28 @@ func (s *CpuGroup) Set(path string, r *configs.Resources) error {
 			period = ""
 		}
 	}
+
+	var burst string
+	if r.CpuBurst != 0 {
+		burst = strconv.FormatUint(r.CpuBurst, 10)
+		if err := cgroups.WriteFile(path, "cpu.cfs_burst_us", burst); err != nil {
+			// Sometimes when the burst to be set is larger
+			// than the current one, it is rejected by the kernel
+			// (EINVAL) as old_quota/new_burst exceeds the parent
+			// cgroup quota limit. If this happens and the quota is
+			// going to be set, ignore the error for now and retry
+			// after setting the quota.
+			if !errors.Is(err, unix.ENOENT) {
+				// this is a special trick for burst feature, the current systemd and low version of kernel will not support it.
+				// So, an `no such file or directory` error would be raised, and we can ignore it .
```

Comment:
I think these two lines comments is for `unix.ENOENT`, you should put the comments in the correct place, so please exchange the comments of `unix.ENOENT` and `unix.EINVAL`.